### PR TITLE
Add attribute and is_feature in condition expression

### PR DESCRIPTION
**Pull request by axwel13**
Hello,

I'm sending you a pull request. It implements the attribute notion available in php8. 
```php
# FooController.php

use Flagception\Bundle\FlagceptionBundle\Attribute\Feature;

#[Feature("feature_123")]
class FooController
{

    #[Feature("feature_789")]
    public function barAction()
    {
    }

    public function fooAction()
    {
    }
}
```

And a function for route expression conditions.

```php
// src/AppBundle/Controller/BlogController.php
// src/Controller/BlogController.php
namespace AppBundle\Controller;

use Symfony\Bundle\FrameworkBundle\Controller\Controller;
use Sensio\Bundle\FrameworkExtraBundle\Configuration\Route;

class BlogController extends Controller
{
    /**
     * @Route("/blog/{page}", condition="is_feature('feature_123')")
     */
    public function listAction($page)
    {
        // ...
    }

    /**
     * @Route("/blog/{slug}")
     */
    public function showAction($slug)
    {
        // ...
    }
}
```

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -25,6 +25,8 @@ Documentation
 * [Usage](docs/usage.md)
 * [Twig flags](docs/twig.md)
 * [Route flags](docs/route.md)
+* [Route condition flags](docs/route_condition.md)
+* [Attribute flags](docs/annotation.md)
```
Comment (migo315) [inline README.md:29]: Shouldn't attribute.md be here?

File: docs/attribute.md [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+    features:      
+        feature_123:
+            default: true
+        
+    # Use annotation? (optional)
+    annotation:
+    
+        # Enable controller annotation (default: false)
+        enable: true
+```
```
Comment (migo315) [inline docs/attribute.md:18]: This config including headline is about annotation and is irrelevant for attributes. Would you like to remove it?

File: composer.json [hunk 3]
```diff
@@ -11,7 +11,7 @@
     }
   ],
   "require": {
-    "php": "^7.1.3 | ^8.0",
+    "php": "^8.0",
```
Comment (migo315) [inline composer.json:14]: Would you also like to remove the PHP 7 tests here https://github.com/playox/flagception-bundle/blob/master/.github/workflows/php.yml?

**Review by migo315 — CHANGES_REQUESTED** [review]
Thank you very much for your contribution! :-)

File: README.md (hunk 1 above)
Comment (axwel13) [reply README.md]: it's corrected !

File: docs/attribute.md (hunk 2 above)
Comment (axwel13) [reply docs/attribute.md:12]: it's corrected !

File: composer.json (hunk 3 above)
Comment (axwel13) [reply composer.json:14]: it's corrected !

File: docs/attribute.md [hunk 4]
```diff
@@ -0,0 +1,37 @@
+Annotation
```
Comment (migo315) [inline docs/attribute.md:1]: Attributes

File: docs/attribute.md [hunk 5]
```diff
@@ -0,0 +1,37 @@
...
+-------------------------
+You can use annotations for checking the feature state in controllers. Just active this in your config:
```
Comment (migo315) [inline docs/attribute.md:3]: Attributes ... and not necessary to activate attributes.

**Review by migo315 — CHANGES_REQUESTED** [review]
Only two minor changes. Then it fits. Great work :-)

File: docs/attribute.md (hunk 4 above)
Comment (axwel13) [reply docs/attribute.md]: Sorry, I quickly copied the annotations documentation. It's corrected.

File: docs/attribute.md (hunk 5 above)
Comment (axwel13) [reply docs/attribute.md]: It's removed.

**Pull request merged**